appinfo/info.xml

```diff
@@ -34,7 +34,7 @@ The rating depends on the installed text processing backend. See [the rating ove
 
 Learn more about the Nextcloud Ethical AI Rating [in our blog](https://nextcloud.com/blog/nextcloud-ethical-ai-rating/).
 	]]></description>
-	<version>4.0.0-beta.2</version>
+	<version>4.1.0-dev.0</version>
```

Comment:
can we use alpha? I think the versions are sorted alphabetically. If we start with dev we can only go to RC and not to alpha or beta